extensions/grpc/runtime/src/main/java/io/quarkus/grpc/GrpcClient.java

```diff
@@ -30,7 +30,7 @@
      */
     String value() default ELEMENT_NAME;
 
-    public final class Literal extends AnnotationLiteral<GrpcClient> implements GrpcClient {
+    final class Literal extends AnnotationLiteral<GrpcClient> implements GrpcClient {
```

Comment:
Why this change?

Reply:
everything is public in an interface anyway

Reply:
Ah, I didn't know it works like this for annotations as well.